lib/posix/signal.c

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2023 Meta
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <errno.h>
+#include <string.h>
+
+#include <zephyr/posix/signal.h>
+
+static inline int signo_word_idx(int signo)
```

Comment:
static inline might not be correct here.

Reply:
converted to macro